src/puppetlabs/puppetdb/config.clj

```diff
@@ -149,7 +149,15 @@
      :temp-usage s/Int
      :memory-usage s/Int
      :max-command-size (pls/defaulted-maybe s/Int (default-max-command-size))
-     :reject-large-commands (pls/defaulted-maybe String "false")}))
+     :reject-large-commands (pls/defaulted-maybe String "false")
+
+     ;; The below config is intended for testing now and isn't documented
+     ;; or expected to be released. Perf testing will confirm if the
+     ;; parameters are needed and what their ideal value would be in our
+     ;; various deployment scenarios
+     :concurrent-writes (pls/defaulted-maybe s/Int 100)
```

Comment:
Update on this, we're not removing it, it turns out to be pretty important, I created ticket PDB-3060 to make this config param "official"